examples/tennis/Tennis.cpp

```diff
@@ -29,7 +27,7 @@ std::filesystem::path resourcesDir()
 ////////////////////////////////////////////////////////////
 int main()
 {
-    std::srand(static_cast<unsigned int>(std::time(nullptr)));
+    std::mt19937 rng = std::mt19937(std::random_device()());
```

Comment:
Ditto

Reply:
Another vexing parse. Either we break with SFML style and use `auto` or we accept this awkward syntax.